src/ncep/SST-NetCDF/make_theia.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+module load netcdf
+
+ifort -c  read_nc_combine_sst_IODA.f  -I/apps/w3lib/2.0.2-intel/lib `nc-config --fflags --flibs --libs`
```

Comment:
Including -I/apps/w3lib/2.0.2-intel/lib a library? 
And Linking with a different actual library -L/scratch3/NCEPDEV/nwprod/lib